### ISSUE TITLE
[REQUEST] Turkish QWERTY

### ISSUE THREAD

**Issue by Atacan-Celikkol** [dslul/openboard#309]
Hi, there is missing dedicated characters in Turkish language. I bring a screenshot from SwiftKey for instance. Can you please add these characters as dedicated buttons? Thank you. 
![Screenshot_20210224-022917](https://user-images.githubusercontent.com/1576322/108923298-9f2abb00-7649-11eb-83b5-607bfd481056.jpg)

**Comment by Yusuf-Kozan** [comment]
Lack of Turkish QWERTY layout is the only reason why I still use Gboard and couldn't switch to OpenBoard.

**Comment by dslul** [comment]
> Lack of Turkish QWERTY layout is the only reason why I still use Gboard and couldn't switch to OpenBoard.

Is something like this adequate? https://github.com/dslul/openboard/commit/f976e1c31c8c01b957d07609597a765f3b037400

![image](https://user-images.githubusercontent.com/6121202/121859490-0b642400-ccf8-11eb-8b7f-9f49c4ebc1f5.png)

**Comment by Yusuf-Kozan** [comment]
The letter between U and O shouldn't have a dot. It's this letter: I

And the letter next to Ş should be İ, not Î

**Comment by Atacan-Celikkol** [comment]
Yes it's pretty similar like that except those two buttons. As you see in my picture, the red one should be "İi" and blue one should be "Iı".

![image](https://user-images.githubusercontent.com/1576322/121864509-a7485c80-cd05-11eb-88c8-a84aa716c980.png)


> > Lack of Turkish QWERTY layout is the only reason why I still use Gboard and couldn't switch to OpenBoard.
> 
> Is something like this adequate? [f976e1c](https://github.com/dslul/openboard/commit/f976e1c31c8c01b957d07609597a765f3b037400)
> 
> ![image](https://user-images.githubusercontent.com/6121202/121859490-0b642400-ccf8-11eb-8b7f-9f49c4ebc1f5.png)

**Comment by ondern** [comment]
https://github.com/rkkr/simple-keyboard

This app provide Turkish-Q layout support. You can fork the code.![184511165-6ad08b60-1ace-4f1a-a29b-5aab5076db69.png](https://user-images.githubusercontent.com/109258788/184544158-d742677c-5ab5-4d83-a27b-8a193b09b159.png)![184511163-fced9696-52a3-4f93-b557-28c2f9d438ae.png](https://user-images.githubusercontent.com/109258788/184544162-c166a1f4-d86a-4b7d-b1a8-90046923ede3.png)![184511158-1ccc6579-a509-46f1-aeaf-b8385804472c.png](https://user-images.githubusercontent.com/109258788/184544166-e1741b8a-4613-480d-b6b3-87baff42bab2.png)![184511156-143d9c4f-9e8c-4cb0-a6ae-9605ba31754c.png](https://user-images.githubusercontent.com/109258788/184544167-ab33770b-015a-410a-9a90-0916b87f124b.png)![184511152-40445d6a-f9fe-4b67-81ee-df846218823b.png](https://user-images.githubusercontent.com/109258788/184544170-a4b9996b-dd85-4a05-a34e-eb44956ef7b7.png)![184511151-0bd56570-af83-4fd1-8b78-9a86a81884ac.png](https://user-images.githubusercontent.com/109258788/184544172-eb65ac86-e2dc-4e2a-8407-891925daf80f.png)

_Originally posted by @ondern in https://github.com/openboard-team/openboard/issues/670#issuecomment-1214401632_

**Comment by mfn77** [comment]
Is there any progress for this?
